packages/core/src/createAtom.ts

```diff
@@ -9,16 +9,16 @@ import {
   Cause,
   createReatomError,
   createTemplateCache,
-  defaultStore,
+  defaultStore as originalStore,
```

Comment:
```suggestion
  defaultStore as originalDefaultStore,
```

Reply:
Also, if you can, rename please `callSafetyDefault` to `originalCallSafety` and `createTemplateCacheDefault` to `originalCreateTemplateCache` [here](https://github.com/artalar/reatom/blob/v2/packages/core/src/createStore.ts)

Reply:
done